portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,29 +42,37 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     int maxComments = Integer.parseInt(request.getParameter("maxComments"));
+    // TODO: sort by timestamp
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
You didn't remove the problematic sorting.

Reply:
weird.. fixed this